.github/workflows/publish-sdist-wheels.yml

```diff
@@ -45,7 +45,15 @@ jobs:
       - name: Build wheels Mac OS
         if: runner.os == 'macOS'
         env:
-          CIBW_BEFORE_BUILD: 'brew update; brew install cmake hdf5'
+          CIBW_BEFORE_BUILD: |
+             brew update
+             brew --version
+             brew install cmake
+             # the HDF5 bottle uses gcc10; https://formulae.brew.sh/formula/hdf5
+             # so we are hit by this: https://github.com/Homebrew/homebrew-core/issues/68866
```

Comment:
whoa that's an important bug from brew...

Reply:
weird/annoying thing is there is a commit saying it's fixed, but it didn't work :(

Reply:
yeah I saw the issue is closed and the commit. But it seems that most of the added lines are not tested. Ex: 
https://github.com/Homebrew/brew/pull/10327/files#diff-86ec75a3d285fc2a3f3e537aa9ec82e30f367fcceb9533cc1e7ac8c357a45187R304